package.json

```diff
@@ -51,6 +52,11 @@
 				"title": "Current file to one other file (not open in editor)",
 				"category": "Meld Diff"
 			},
+			{
+				"command": "meld-diff.diffWithClipboard",
+				"title": "Diff with clipboard",
```

Comment:
I would prefer according to `meld-diff.diffFromFileList` to use `"Compare with clipboard"` as title.

Reply:
Thanks for your constructive feedback. Renaming the title is also ok for me.

I really like the idea of supporting additional diff tools. Using settings options is definitively a good idea.